DSNP-Wallet/Source/Common/ExtrinsicManager/FrequencyChain.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  FrequencyChain.swift
+//  DSNP-Wallet
+//
+//  Created by Ryan Sheh on 2/22/23.
+//
+
+import Foundation
+import RobinHood
+import BigInt
+
+enum Currency: BigUInt {
+    case dollar = 100000000 //one token = 100000000 units
+    case cent = 1000000
+    case millicent = 1000
+    case unit = 1
+}
+
+struct FrequencyChain {
+    static let shared = FrequencyChain()
+    let id = "496e2f8a93bf4576317f998d01480f9068014b368856ec088a63e57071cd1d49"
```

Comment:
where does this id come from?

Reply:
The ID is the genesis hash for the chain.